Gemfile.lock

```diff
@@ -154,7 +173,10 @@ DEPENDENCIES
   kramdown
   liquid
   listen
+  rake
 
 RUBY VERSION
    ruby 2.5.5p157
 
+BUNDLED WITH
```

Comment:
This may break the website generation on the builder. I don't recall the available version but most probably inferior compared the one on your system. Also it is not needed (but added automatically by bundler comands), so please remove it.

Reply:
ok